src/Razor/src/Microsoft.VisualStudio.RazorExtension/SyntaxVisualizer/SyntaxVisualizerControl.xaml.cs

```diff
@@ -141,6 +178,39 @@ public void ShowGeneratedCode()
             return;
         }
 
+        EnsureInitialized();
+
+        if (_languageServerFeatureOptions.UseRazorCohostServer)
+        {
+            if (_fileUriProvider.TryGet(_activeWpfTextView.TextBuffer, out var hostDocumentUri))
```

Comment:
Combine this if with the outer if. Put each condition on its own line though, the `&&` at the end of the first